packages/webpack-extraction-plugin/src/GriffelCSSExtractionPlugin.test.ts

```diff
@@ -28,13 +28,16 @@ async function compileSourceWithWebpack(
 }> {
   const defaultConfig: webpack.Configuration = {
     context: __dirname,
-    entry: entryPath,
+    entry: {
+      bundle: entryPath,
+    },
 
-    mode: 'development',
+    mode: 'production',
```

Comment:
should we have tests for both modes? 🤔

Reply:
I was thinking about that, but all our partners use it only in production mode.